packages/firestore/src/util/log.ts

```diff
@@ -59,10 +59,10 @@ export function setLogLevel(newLevel: LogLevel): void {
 }
 
 export function debug(tag: string, msg: string, ...obj: unknown[]): void {
-  if (logClient.logLevel <= FirebaseLogLevel.DEBUG) {
-    const args = obj.map(argToString);
-    logClient.debug(`Firestore (${SDK_VERSION}) [${tag}]: ${msg}`, ...args);
-  }
+  //if (logClient.logLevel <= FirebaseLogLevel.DEBUG) {
+  const args = obj.map(argToString);
+  console.debug(`Firestore (${SDK_VERSION}) [${tag}]: ${msg}`, ...args);
```

Comment:
Intended?

Reply:
No. Sorry that is just my preferred way of debugging :/